server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/MissingValueIT.java

```diff
@@ -54,161 +54,186 @@ protected void setupSuiteScopeCluster() throws Exception {
-            assertEquals(1, terms.getBucketByKey("bar").getDocCount());
-
-            response = prepareSearch("idx").addAggregation(terms("my_terms").field("str").missing("foo")).get();
-            assertNoFailures(response);
-            terms = response.getAggregations().get("my_terms");
-            assertEquals(1, terms.getBuckets().size());
-            assertEquals(2, terms.getBucketByKey("foo").getDocCount());
+            assertNoFailuresAndResponse(
+                prepareSearch("idx").addAggregation(terms("my_terms").field("str").executionHint(mode.toString()).missing("bar")),
+                response -> {
+                    assertNoFailures(response);
+                    Terms terms = response.getAggregations().get("my_terms");
+                    assertEquals(2, terms.getBuckets().size());
+                    assertEquals(1, terms.getBucketByKey("foo").getDocCount());
+                    assertEquals(1, terms.getBucketByKey("bar").getDocCount());
+                }
+            );
+            assertNoFailuresAndResponse(prepareSearch("idx").addAggregation(terms("my_terms").field("str").missing("foo")), response -> {
+                Terms terms = response.getAggregations().get("my_terms");
+                assertEquals(1, terms.getBuckets().size());
+                assertEquals(2, terms.getBucketByKey("foo").getDocCount());
+            });
         }
     }
 
     public void testLongTerms() {
-        SearchResponse response = prepareSearch("idx").addAggregation(terms("my_terms").field("long").missing(4)).get();
-        assertNoFailures(response);
-        Terms terms = response.getAggregations().get("my_terms");
-        assertEquals(2, terms.getBuckets().size());
-        assertEquals(1, terms.getBucketByKey("3").getDocCount());
-        assertEquals(1, terms.getBucketByKey("4").getDocCount());
-
-        response = prepareSearch("idx").addAggregation(terms("my_terms").field("long").missing(3)).get();
-        assertNoFailures(response);
-        terms = response.getAggregations().get("my_terms");
-        assertEquals(1, terms.getBuckets().size());
-        assertEquals(2, terms.getBucketByKey("3").getDocCount());
+        assertNoFailuresAndResponse(prepareSearch("idx").addAggregation(terms("my_terms").field("long").missing(4)), response -> {
+            Terms terms = response.getAggregations().get("my_terms");
+            assertEquals(2, terms.getBuckets().size());
+            assertEquals(1, terms.getBucketByKey("3").getDocCount());
+            assertEquals(1, terms.getBucketByKey("4").getDocCount());
+        });
+        assertNoFailuresAndResponse(prepareSearch("idx").addAggregation(terms("my_terms").field("long").missing(3)), response -> {
+            assertNoFailures(response);
+            Terms terms2 = response.getAggregations().get("my_terms");
+            assertEquals(1, terms2.getBuckets().size());
+            assertEquals(2, terms2.getBucketByKey("3").getDocCount());
```

Comment:
IntelliJ being silly with its variable names I think

```suggestion
            Terms terms = response.getAggregations().get("my_terms");
            assertEquals(1, terms.getBuckets().size());
            assertEquals(2, terms.getBucketByKey("3").getDocCount());
```

Reply:
I admit I did this by hand :/ no clue why ... sorry!